elasticdl/python/collective_ops/communicator.py

```diff
@@ -16,10 +18,15 @@
 
 
 class CollectiveCommunicator(object):
-    def __init__(self):
+    def __init__(self, svc_name=None):
```

Comment:
What is this `svc_name`? Current worker's service name, or FtLib service?

Reply:
FTLib k8s service. I’ll work on adding that service in a follow-up PR

Reply:
`getaddrinfo` of this `svc_name` will return a list of all worker ip info?

Reply:
Yes, this way the consensus method knows what addresses are known.